### PR TITLE
[dy] Run all blocks

### PR DESCRIPTION
**Pull request by dy46**
# Summary

Don't check block status when running a pipeline.

# Tests

tested locally

cc: @wangxiaoyou1993

### CODE REVIEW

File: mage_ai/__init__.py [hunk 1]
```diff
@@ -121,11 +121,10 @@ def clean(
...
 # --------------- Data preparation methods --------------- #
 
-
 def run(pipeline_uuid: str, project_path: str = None, **global_vars) -> None:
     project_path = os.getcwd() if project_path is None else os.path.abspath(project_path)
     sys.path.append(os.path.dirname(project_path))
     pipeline = Pipeline(pipeline_uuid, project_path)
     asyncio.run(
-        pipeline.execute(analyze_outputs=False, global_vars=global_vars, update_status=False)
+        pipeline.execute(analyze_outputs=False, global_vars=global_vars, update_status=True)
```
Comment (wangxiaoyou1993) [inline mage_ai/__init__.py:129]: I purposely set it to False because writing to metadata.yaml throws permission error when I run it in Airflow.

Comment (dy46) [reply mage_ai/__init__.py:129]: I see

Comment (dy46) [reply mage_ai/__init__.py:129]: Setting this as false can lead to some errors when running the pipeline if the blocks haven't been executed yet

Comment (wangxiaoyou1993) [reply mage_ai/__init__.py:129]: Should it just run all the blocks?

Comment (dy46) [reply mage_ai/__init__.py:129]: yeah it should, but there's some validation that will fail if the status is not set. i'm looking to see if we can skip that validation

**Review by wangxiaoyou1993 — APPROVED** [review]

**Pull request merged**